atomic_reactor/tasks/sources.py

```diff
@@ -44,15 +53,17 @@ class SourceBuildTask(plugin_based.PluginBasedTask):
             {"name": "compress", "args": {"load_exported_image": True, "method": "gzip"}},
             {"name": "tag_and_push"},
         ],
+    )
+
+
+class SourceBuildExitTask(SourceBuildBaseTask):
+    """Source container build task."""
+
+    plugins_def = plugin_based.PluginsDef(
         exit=[
             {"name": "verify_media", "required": False},
             {"name": "koji_import_source_container"},
             {"name": "koji_tag_build"},
             {"name": "store_metadata"},
         ],
     )
```

Comment:
This won't work, because the `workflow.build_docker_image()` method cannot handle when there are 0 buildstep plugins.

@mkosiarc, you encountered this while working on the build task, do you have a solution yet? If yes, could it be a standalone PR?

Reply:
I don't have it yet, but will work on it soon. If I finish this and test that it works, I can create a standalone PR.

Reply:
Ack, just to make sure @tkdchen and you don't do the same work

Reply:
It won't work even if you move the plugins from `exit` to `prebuild` :disappointed: 

At least the plugins will get executed this way, but after that the finally task will fail with "No appropriate build step"